pyproject.toml

```diff
@@ -0,0 +1,142 @@
+[build-system]
+requires = ["setuptools>=61.0", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "wiki2beamer"
+version = "0.10.0"
+description = "Create LaTeX-beamer presentations with a wiki syntax"
+readme = "README.md"
+license = {text = "GPL-2.0-or-later"}
+requires-python = ">=3.7"
+authors = [
+    {name = "Michael Rentzsch", email = "michael.rentzsch@repc.de"},
+    {name = "Kai Dietrich", email = "mail@cleeus.de"},
+]
+maintainers = [
+    {name = "Valentin Haenel", email = "valentin@haenel.co"},
+]
+keywords = ["latex", "beamer", "presentation", "wiki"]
+dependencies = []
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Environment :: Console",
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: GNU General Public License v2 or later (GPLv2+)",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
```

Comment:
this list above doesn't match the versions in the github workflow for testing.

Would suggest to use: 3.9 - 3.13 as these are the currently supported python versions:

https://devguide.python.org/versions/

Reply:
Fixed

Reply:
Fixed.  not sure why it still shows here as unchanged.